lib/Runtime/Base/CallInfo.h

```diff
@@ -77,7 +77,7 @@ namespace Js
 
         bool HasExtraArg() const
         {
-            return (this->Flags & CallFlags_ExtraArg) || this->HasNewTarget();
+            return  CallInfo::HasExtraArg(this->Flags);
```

Comment:
nit: extra space

Reply:
Uh. Did not notice. Will try fixing in some next change.

Reply:
SGTM